src/Parser/Core/Modules/Items/Prydaz.js

```diff
@@ -22,7 +24,11 @@ class Prydaz extends Module {
   item() {
     return {
       item: ITEMS.PRYDAZ_XAVARICS_MAGNUM_OPUS,
-      result: this.owner.formatItemHealingDone(this.healing),
+      result: (
+        <dfn data-tip={`The total damaged absorbed by Archimonde's Hatred Reborn was ${this.owner.formatItemAbsorbDone(this.healing)}.`} >
```

Comment:
Also, this is Prydaz module, why is there even a comment about Archimonde's Hatred Reborn (tanking legendary trinket I believe)? Maybe a missed rename or something?